GoogleSignIn/Sources/GIDScopes.m

```diff
@@ -29,6 +30,10 @@ static BOOL hasEmail(NSString *scope) {
   return [scope isEqualToString:kEmailScope] || [scope isEqualToString:kOldEmailScope];
 }
 
+static BOOL hasOpenID(NSString *scope) {
```

Comment:
The remaining changes in this file seem to be unused now.

Reply:
Done.